package.json

```diff
@@ -32,7 +32,9 @@
 		"url": "https://github.com/sparksuite/w3c-css-validator/issues"
 	},
 	"homepage": "https://sparksuite.github.io/w3c-css-validator/",
-	"dependencies": {},
+	"dependencies": {
+		"form-data": "^4.0.0"
+	},
```

Comment:
Is this absolutely required? If we add this, we'll have to change the "Zero dependencies" line in the README.

Reply:
We need some way to build the form data in node environments when submitting via direct input, I think we need a third party package to accomplish that, if we don't want to build it ourselves?

Reply:
We might be able to get away with this: https://stackoverflow.com/a/70548249/11753100